docs/reference/migration/migrate_6_0/scripting.asciidoc

```diff
@@ -33,4 +33,8 @@ The `lang` variable can no longer be specified as part of a request that uses a
 script otherwise an error will occur.  Note that a request using a stored script is
 different from a request that puts a stored script.  The language of the script has
 already been stored as part of the cluster state and an `id` is sufficient to access
-all of the information necessary to execute a stored script.
\ No newline at end of file
+all of the information necessary to execute a stored script.
+
+==== Stored template apis removed
+
+The PUT, GET and DELETE _template apis have been removed. Store templates with the stored scripts apis instead.
```

Comment:
Again, this is not the `/_template` APIs, this is `/_search/template` APIs.

I do think it'd be extremely beneficial to have a "if you did this, now do this"
part in these docs as a quick example

Here's an example if you want to use it:

If you previously did:

```json
PUT /_search/template/my_template
{
  "query": {
    "match": {
      "f1": "{{f1}}"
    }
  }
}
```

You should do now do

```json
PUT /_scripts/my_template
{
  "script": {
    "lang": "mustache",
    "source": {
      "query": {
        "match": {
          "f1": "{{f1}}"
        }
      }
    }
  }
}
```